integration-test/src/integrationTest/kotlin/com/improve_future/harmonica/integration/PostgreSqlMigrationTest.kt

```
@@ -0,0 +1,60 @@
+            user = TestDb.postgresUser
+            password = TestDb.postgresPassword
+        }.use { connection ->
+            connection.transaction {
+                object : AbstractMigration() {
+                    override fun up() {
+                        createTable("postgres_table") {
+                            varchar("name")
+                        }
+                    }
+
+                    override fun down() {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)